source/ocsp-support/ocsp-support.rst

```diff
@@ -256,7 +277,11 @@ expose an option to enable/disable certificate revocation checking on a
 per MongoClient basis.
 
 1. Driver MUST enable OCSP support (with stapling if possible) when
-   certificate revocation checking is enabled.
+   certificate revocation checking is enabled UNLESS their driver
```

Comment:
"UNLESS" is not a special term from RFC 2119, so I don't think this needs to be capitalized. Feel free to add italics for emphasis.

Reply:
Ah true, fixed (went with a bold).